internal/format/asciidoc_document.go

```diff
@@ -17,6 +17,23 @@ const (
 		{{- end -}}
 	{{ end -}}
 	`
+	asciidocDocumentResourcesTpl = `
+	{{- if .Settings.ShowResources -}}
+		{{ indent 0 "=" }} Resources
+		{{ if not .Module.Resources }}
+			No resources.
+		{{ else }}
+			The following resources are used by this module:
+			{{ range .Module.Resources }}
+				{{ if eq (len .URL) 0 }}
+				- .FullType
```

Comment:
Brackets are missing.

```suggestion
				- {{ .FullType }}
```

Reply:
🤦